parquet-column/src/main/java/org/apache/parquet/column/statistics/Float16Statistics.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.parquet.column.statistics;
+
+import org.apache.parquet.schema.PrimitiveType;
+
+public class Float16Statistics extends BinaryStatistics {
```

Comment:
If there is no override, do we actually need this subclass?

Reply:
```
  Float16Statistics(PrimitiveType type) {
    super(type);
  }
```
 we need it here

Reply:
I mean, why not directly use `BinaryStatistics` if `Float16Statistics` does not add any specific logic to it? Is anywhere relying on a `instanceof Float16Statistics` check?

Reply:
you are correct, we can use BinaryStatistics directly and we don't need to have Float16Statistics.